.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,10 @@
 ### PR checklist
 
-- [ ] Provide an overview of the changes you're making and explain why you're proposing them, ideally in the form of a changelog (template below)
-- [ ] Include `Fixes #NNN` somewhere in the description if this PR addresses an existing issue.
+- [ ] Provide an overview of the changes you're making and explain why you're proposing them, ideally in the form of a changelog (use `scriv create` or add `CHANGELOG-missing` label to generate one from title).
```

Comment:
```suggestion
- [ ] Provide an overview of the changes you're making and explain why you're proposing them, ideally in the form of a changelog snippet (use `scriv create` or add the `CHANGELOG-missing` label to this pull request in order to generate one from its title).
```

Reply:
Or, if we want to disassociate PR description and changelog entry, this could be:
```
- [ ] Provide an overview of the changes you're making and explain why you're proposing them
- [ ] Create a changelog snippet (use `scriv create` locally and include the generated file in the pull request, or add the `CHANGELOG-missing` label to this pull request in order to have a snippet generated from its title; see [scriv](https://scriv.readthedocs.io/)).
```
but the above is also fine.

Reply:
Agree. That would allow a "ask for forgiveness" approach by contributors, checking off the first one only and allow "us" to fix for the second if they didn't.

Reply:
I like @mslw proposed separation, and reordered to list label first so then scriv ref comes right on